recipe/meta.yaml

```diff
@@ -12,34 +12,47 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  Skip: True  # [py<38]
```

Comment:
```suggestion
```
it declares support for older versions: https://github.com/Rockhopper-Technologies/jinxed/blob/1.2.0/setup.py#L54